src/iso-registry-client/src/main/webapp/WEB-INF/views/registry/registers/gcp/infosrc_panel.html

```diff
@@ -87,7 +87,7 @@
 					<div class="modal-header">
 						<h4 th:text="#{popup.informationSource.header}"></h4>
 					</div>
-					<div th:if="!${isProposal} or ${isReadOnly}" class="modal-body" th:object="${item.informationSource[__${rowstat.index}__]}">
+					<div th:if="!${isProposal} or ${isReadOnly}" class="modal-body" th:object="${proposal.informationSource[__${rowstat.index}__]}">
 						<div th:include="registry/registers/gcp/infosrc_panel_content :: informationSourcePanelContent(${rowstat.index})"/>
 					</div>
 					<div th:if="${isProposal} and !${isReadOnly}" class="modal-body" th:object="${proposal.informationSource[__${rowstat.index}__]}" th:with="isReadOnly='true'">
```

Comment:
```suggestion
					<div th:if="${isProposal}" class="modal-body" th:object="${proposal.informationSource[__${rowstat.index}__]}" th:with="isReadOnly='true'">
```

Reply:
@phuonghuynh My old code here really makes not much sense. I think this is also what's causing that editing  citations is not possible in a clarification. For this PR it's not necessary to change it, though.